geogen: tighten bounds of SRID 0 for coordinate generation

otan opened the pull request:
[`math.MaxFloat64 + math.MaxFloat64 =
Inf`](https://play.golang.org/p/kU9uLo5SCyo), so this was generating
invalid coordinate values. Loosen to float32 bounds.

Resolves #52290. 
Resolves #52291.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/52340)
<!-- Reviewable:end -->

otan committed:
geogen: tighten bounds of SRID 0 for coordinate generation

[`math.MaxFloat64 + math.MaxFloat64 =
Inf`](https://play.golang.org/p/kU9uLo5SCyo), so this was generating
invalid coordinate values. Loosen to float32 bounds.

Release note: None

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/52340#-:-MDtyYqr5Ku4vz_JFWiA:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

otan commented:
tftr!

bors r=sumeerbhola
